### ISSUE TITLE
cannot compile this under osx Big Sur (beta)

### ISSUE THREAD

**Issue by eerotvolotinen** [etlegacy/etlegacy#1512]
./easybuild.sh -64

..

 "_vsnprintf", referenced from:
      _PC_SourceError in bg_misc.c.o
      _Com_sprintf in q_shared.c.o
      _COM_ParseError in q_shared.c.o
      _Com_DPrintf in ui_atoms.c.o
      _Com_Printf in ui_atoms.c.o
      _Com_Error in ui_atoms.c.o
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by jeannekamikaze** [comment]
I am new to this project, but did you try this?

```
Depending on what brew version you're using (mostly older ones), you have to specify brew install --universal to get both 32bit and 64bit libs.
```
Either you don't have the 32-bit libraries installed or your build can't find them.

**Comment by eerotvolotinen** [comment]
tried again:

got error:

/Users/eerovolotinen/etsources/etlegacy/src/sys/sys_main.c:1012:23: error: implicit declaration of function 'needsOSXQuarantineFix' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
                quarantine_status = needsOSXQuarantineFix();
                                    ^
1 error generated.
make[2]: *** [CMakeFiles/etl.dir/src/sys/sys_main.c.o] Error 1
make[1]: *** [CMakeFiles/etl.dir/all] Error 2
make: *** [all] Error 2
Exiting!

**Comment by zedar187** [comment]
@eerotvolotinen can you try compiling by using the additional osx parameter?

`$ ./easybuild.sh -64 --osx=10.15`

I haven't got Big Sur yet, so i'm kinda blindfolded in this case.

**Comment by zedar187** [comment]
Closed in favor of #1523.
